packages/aws-cdk/lib/api/bootstrap/bootstrap-environment.ts

```diff
@@ -139,3 +158,8 @@ export class Bootstrapper {
     }
   }
 }
+
+/**
+ * Magic parameter value that will cause the bootstrap-template.yml to NOT create a CMK but use the default keyo
+ */
+const USE_AWS_MANAGED_KEY = '-';
```

Comment:
`~` Any reason not to make this a little less magical? Like 
```suggestion
const USE_AWS_MANAGED_KEY = 'USE_AWS_MANAGED_KEY';
```

Reply:
I guess not!